index.html

```diff
@@ -0,0 +1,201 @@
+    <title>cars show information</title>
+  </head>
+  <body>
+   
+    <header>
+      <div class="top-section">
+          <ul class="bars">
+            <li>
+              <i class="fab fa-facebook-f"></i>
+            </li>
+            <li>
+              <i class="fab fa-twitter"></i>
+            </li>
+            <li>
+              <h4>English</h4>
+            </li>
+            <li>
+              <h4>My Page</h4>
+            </li>
+            <li>
+              <h4>Logout</h4>
+            </li>
+          </ul>
+        </div>
+      <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"
+            ><img src="img/pakwheels.png" alt="plane" class="plane"
+          /></a>
+          <div class="menu-bar nav" id="menu">
+            <i class="fas fa-bars fa-3x"></i>
+          </div>
+          <div class="cancel" id="close">
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+
+          <div class="company">
+            <a href="index.html">car Circle</a>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    <section class="section-one">
+      <div class="section-one-container">
+        <h2 class="intro">
+         THE CAR SHOW
+        </h2>
+        
+        <h1 class="heading">PAKWHEELS CAR SHOW 2022</h1>
+        <div class="paragraph1">
+          <p>
+            Are you ready for Car Mela! It's happening again this weekend.
+            Click here to register your car now :  
+            Venue: 2F2F Karting, Lake view park, Islamabad
+            Date : 13th March,2022
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="section-two" id="program">
+      <div class="section-two-container">
+        <h2 class="heading2">Main Program</h2>
+        <hr />
+        <div class="program">
+          <div class="old-cars stacks">
+            <img class="icons" src="img/pak2.svg" alt="old-cars" />
+            <h2>Old cars</h2>
+            <p>
+              Classic pieces kept things simple. They are easy to work on, there are no computers, no complicated wiring, and there is plenty of room for comfort.
+            </p>
+          </div>
+
+          <div class="sports-cars stacks">
+            <img class="icons" src="img/pak1.svg" alt="sports-cars" />
+            <h2>Sports cars</h2>
```

Comment:
- [ ] I recommend using `grid` CSS property to evenly layout columns in the `Main program` section :

Your design  | With Grid
------------- | -------------
<img width="412" alt="Screenshot 2022-03-16 at 18 30 47" src="https://user-images.githubusercontent.com/77038610/158613488-36da7733-dfa1-4f61-95f1-678f177a1975.png"> | <img width="410" alt="Screenshot 2022-03-16 at 18 27 16" src="https://user-images.githubusercontent.com/77038610/158612712-c7788e62-0502-4d1b-93ad-44953e134bf8.png">

Eg. for mobile screens: 
```
.stacks {
    display: grid;
    grid-template-columns: 1fr 1fr 3fr;
}
```